pubnub/pubnub_core.py

```diff
@@ -27,7 +27,7 @@
 from .endpoints.access.audit import Audit
 from .endpoints.access.grant import Grant
 from .endpoints.access.grant_token import GrantToken
-from .endpoints.access.revoke import Revoke
+from .endpoints.access.revoke import RevokeToken
```

Comment:
shouldn't `endpoints.access.revoke` be `endpoints.access.revoke_token` similar to grant_token?

Reply:
good idea 👌